changes/ce/feat-11487.en.md

```diff
@@ -0,0 +1,3 @@
+The default number ob bcrypt rounds for pasword hashing is reduced to a more usable value by default (5).
```

Comment:
Where does this number come from?

Reply:
It's from the task https://emqx.atlassian.net/browse/EMQX-10399

The security  recomendation is to use a value that evaluates ~250ms. 

On a modern commodity server CPU, I have about 120ms for 10 log rounds.

My understanding of lowering the round number is the following:

* The whole MQTT context. Even with parallel bcrypt evaluation, secure values allow only few _tens_ of clients to connect per second without occupying the whole node.
* The excessiveness of security compared to other proposed algorithms.

However, I agree that we should strongly motivate the security weakening.

@zmstone, WDYT?

Reply:
I think we should keep default to 10.

Reply:
✅